modules/ROOT/pages/migration-core-foreach.adoc

```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Little changed between For Each scope for Mule 3 and Mule 4 except for the use of DataWeave 2 as the expression language. This little change however has a huge impact in terms of usability of the component.
+The For Each scope didn't change much from Mule runtime engine (Mule) 3 to Mule 4, except for the use of DataWeave 2 as the expression language. This one change has a large impact in terms of usability of the component.
```

Comment:
Should the For Each be `foreach` ?

Reply:
It is (inconsistently), so I'm not touching this one.